modules/apps/frontend-theme/frontend-theme-unstyled/src/main/resources/META-INF/resources/_unstyled/templates/init.ftl

```diff
@@ -268,7 +268,7 @@
 </#if>
 
 <#if htmlTitle??>
-	<#assign html_title = htmlUtil.escape(htmlTitle) />
+	<#assign html_title = htmlTitle />
```

Comment:
Hey @jgarcialr, not a security expert here, but you should always try to escape as close to the context as possible, so we should be doing exactly the opposite.

Escaping is context-aware, so we should escape when we know in which context we're using the value. For instance, we need different escaping algorithms depending if we're printing this in HTML or in an HTML attribute or in a JS string. All those are reigned by different rules and it's not something a Contributor high up could know about.

Additionally, someone could add their own contributor to override `htmlTitle` and forget to escape it opening the door to possible xss attacks.

The fact that this variable is called `htmlTitle` can somehow act as a hint that this is an html-safe value (already escaped) but it would mean anyone using it elsewhere (an attribute, or passing it through JS) could find themselves vulnerable.

Ideally, I think we'd want to stop escaping in the contributor and start escaping closer to usage. I think [LayoutSEOLinkManagerImpl](https://github.com/liferay-frontend/liferay-portal/blob/master/modules/apps/layout/layout-seo-service/src/main/java/com/liferay/layout/seo/internal/LayoutSEOLinkManagerImpl.java) is doing too much escaping for our own good, especially since none of those methods are unequivocally signalling that content is escaped in a meaningful way and thus expected to be used in a particular context.